target/linux/x86/base-files/etc/board.d/02_network

```diff
@@ -11,6 +11,10 @@ board_config_update
 board="$(cat /tmp/sysinfo/board_name)" 2>/dev/null
 
 case "$board" in
+pc-engines-apu2)
+	ucidef_set_interfaces_lan_wan "eth1" "eth0"
+	ucidef_set_interface_raw "opt" "eth2"
```

Comment:
Should I change the above-cited part to just one line
```
        ucidef_set_interfaces_lan_wan "eth1 eth2" "eth0"
```
?

Or drop that patch entirely and submit only the bits I can test, leaving @riptidewave93 to reach consensus, test, and submit the APU2 part?